helper/validation.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+function image_check($pic){
+    
+    $uploadOk = 1;
+    
+    if($_FILES[$pic]['error']==0){ // $FILES['name']['error'] is 0 if successfully uploaded
+    
+        $extension = pathinfo( basename($_FILES[$pic]["name"]))['extension'];
```

Comment:
Use single quote(') and make consistency in project
 $extension = pathinfo(basename($_FILES[$pic]['name']))['extension'];

DO changes in all places where ever is required